orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/Stage.java

```diff
@@ -435,7 +436,7 @@ public Task taskById(String taskId) {
     try {
       return objectMapper.readValue(data, type);
     } catch (IOException e) {
-      throw new RuntimeException("Could not convert " + new String(data) + " to " + type.getSimpleName());
+      throw new RuntimeException("Could not convert " + new String(data, UTF_8) + " to " + type.getSimpleName(), e);
```

Comment:
Did you see something that prompted this?

Reply:
noticed a compilation warning as I was testing, so I figured I'd fix